osu.Game/Overlays/NewsOverlay.cs

```diff
@@ -67,7 +67,26 @@ private void load()
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            article.BindValueChanged(onArticleChanged, true);
+            article.BindValueChanged(onArticleChanged);
```

Comment:
should probably be renamed to `updateDisplay(), then `PopIn` would call it. better than article change trigger.